environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -144,11 +144,29 @@ private ValidationResult validateEntitlement(ValidationResultBuilder validationR
                         build();
             }
         }
+        String encryptionKeyResourceGroupName = azureResourceEncryptionParametersDto.getEncryptionKeyResourceGroupName();
```

Comment:
Can we have an individual method for validating of encryption key resource group name?

Reply:
Moved the validation of encryption resources to a separate method: `validateEncryptionParameters`